files/client.ini

```diff
@@ -0,0 +1,3 @@
+[client]
+homedir=~/.securedrop_client
+use_securedrop_proxy=True
```

Comment:
Nit: missing newline at EOF

Reply:
Yup, will add this.

Reply:
I will wait for all the other issues to be discussed and resolved, I will fix at the very end :)